runtime/bcutil/ClassFileOracle.cpp

```diff
@@ -835,11 +835,18 @@ ClassFileOracle::computeSendSlotCount(U_16 methodIndex)
 			while ((index < count) && ('[' == bytes[index])) {
 				++index;
 			}
+#if defined(J9VM_OPT_VALHALLA_VALUE_TYPES)
+			if ((index >= count) || !(('L' == bytes[index]) || ('Q' == bytes[index]))) {
```

Comment:
It is easier to read as `(index >= count) || ('L' != bytes[index]) || ('Q' != bytes[index])`

Reply:
I believe the correct logic, in a more readable way, would be:
`(index >= count) || (('L' != bytes[index]) && ('Q' == bytes[index]))`
The desired logic is as follows:
> If (index >= count) or (the next character isn't a Q or L)